CGPs/0006.md

```diff
@@ -32,7 +32,7 @@ To not put too much selling pressure on CELO, the rebalancing will happen gradua
 
 ## Verification
 
-1. Confirm proposal steps: run `celocli governance:view --proposalID x` // TODO add proposal ID once submitted proposal
+1. Confirm proposal steps: run `celocli governance:view --proposalID 6`
```

Comment:
```suggestion
1. Confirm proposal steps: run `celocli governance:view --proposalID 7`
```
This would be proposal ID 7 IIUC

Reply:
This is a bit out of place, sorry for the confusion.  I'm just editing the previous proposal.

Reply:
Oh sure that makes sense, thanks for clarifying.